src/structures/Modal.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const BaseMessageComponent = require('./BaseMessageComponent');
+const Util = require('../util/Util');
+
+/**
+ * Represents an modal to be shown
```

Comment:
english
```suggestion
 * Represents a modal (Form) to be shown through an interaction
```

Reply:
Not a fan of this wording, no. I'll fix the "a" though.

Reply:
I'm just using the same sentence they used in the announcement, without that it seems like it's lacking something